spec/features/dashboards/index_spec.rb

```diff
@@ -44,43 +57,4 @@
       expect(page).to have_link("New to AMAFlix? Register Here.", :href=>"/register" )
     end
   end
-
```

Comment:
It looks like maybe you deleted it by accident? It's in my branch so deleting it from here is fine.